docker/ccf_app.virtual

```diff
@@ -0,0 +1,20 @@
+# Build
+FROM mcr.microsoft.com/ccf/app/dev:2.0.7-sgx as builder
+COPY . /src
+RUN mkdir -p /build/
+WORKDIR /build/
+RUN CC="/opt/oe_lvi/clang-10" CXX="/opt/oe_lvi/clang++-10" cmake -GNinja /src && ninja
+
+# Run
+FROM mcr.microsoft.com/ccf/app/run:2.0.7-sgx
+
+COPY --from=builder /build/libccf_app.virtual.so /app/
+COPY --from=builder /opt/ccf/bin/*.js /app/
+COPY --from=builder /opt/ccf/bin/keygenerator.sh /app/ 
+COPY ./config/cchost_config_virtual.json /app/
+WORKDIR /app/
+RUN /app/keygenerator.sh --name member0 --gen-enc-key
+
+EXPOSE 8080/tcp
```

Comment:
I tried this but for some reason when running as container, connection gets refused. Could be an error on my end.

Reply:
Which address did you use to connect to the CCF service from outside the container? You should use the address of the container, that you can find out with `docker inspect <name_of_container>`, e.g. `curl https://172.17.0.2:8000/node/commit`,